File: dogs/backend/src/main/java/com/example/demo/entity/Dog.java
package com.example.demo.entity;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Data;

@Data
@Entity
@Table(name = "dogs")
public class Dog {
	@Id
	private String id;
	private String ownerId;
	private String name;
	private String kind;
	private String sex;
	private String age;
	private String imagePath;
}

File: dogs/angular-front/src/app/models/dog.ts
export class Dog {
  id: string;
  name: string;
  age: string;
  kind: string;
  sex: string;
  image: string;

  constructor(id: string, name: string, age: string, kind: string, sex: string, image: string){
    this.id = id;
    this.name = name;
    this.age = age;
    this.kind = kind;
    this.sex = sex;
    this.image = image;
  };
}